assets/src/blocks/Submenu/submenuFunctions.js

```diff
@@ -0,0 +1,79 @@
+// Map for old attribute 'submenu_style'
+const SUBMENU_STYLES = {
+  1: 'long',
+  2: 'short',
+  3: 'sidebar'
+};
+
+export const getSubmenuStyle = (className, submenu_style) => {
+  let style = submenu_style ? SUBMENU_STYLES[submenu_style] : 'long';
+  if (className && className.includes('is-style-')) {
```

Comment:
If you do the check first, you don't need a variable and it's a bit more clear what's happening:
```
  if (className && className.includes('is-style-')) {
    return className.split('is-style-')[1];
  }
  return submenu_style ? SUBMENU_STYLES[submenu_style] : 'long';
```